cirq-core/cirq/sim/classical_simulator.py

```diff
@@ -117,12 +117,24 @@ def _act_on_fallback_(self, action, qubits: Sequence['cirq.Qid'], allow_decompos
 
         Raises:
             ValueError: If initial_state shape for type np.ndarray is not equal to 1.
-                        If gate is not one of X, CNOT, SWAP, CCNOT, or a measurement.
+                        If gate is not one of X, SWAP, a controlled version of X or SWAP, or a measurement.
```

Comment:
whoops, for some reason when I ran the linter locally it didn't flag on this (but the CI shows this line is too long). what's the correct indentation for breaking up a line inside of this Raises/ValueError block?

Reply:
line length should be <= 100

Reply:
Thanks, I am aware, that's why I made my comment. I'm not asking about the length it should be, I'm asking about how the indentation should work, if I break up the line that is currently too long. Do I indent at the same level as  "If gate..." or indent one tab further?

Reply:
that is more or less a style thing, whichever looks better. I would say one tab further and if it looks unnatural we can remove the extra tab

Reply:
ah, sounds good. I wasn't sure whether that would matter to whatever you use to parse the docstrings and turn them into documentation (is it sphinx? I somehow couldn't find any information about it in the dev documentation)

Reply:
fixed!